src/bitcoin-cli.cpp

```diff
@@ -642,7 +642,7 @@ class NetinfoRequestHandler : public BaseRequestHandler
         "  send     Time since last message sent to the peer, in seconds\n"
         "  recv     Time since last message received from the peer, in seconds\n"
         "  txn      Time since last novel transaction received from the peer and accepted into our mempool, in minutes\n"
-        "           \"*\" - the peer requested we not relay transactions to it (relaytxes is false)\n"
+        "           \"*\" - we do not relay transactions to this peer (relaytxes is false)\n"
```

Comment:
Sorry for the late comment, but where did this behaviour change? Or was the previous description always wrong?

I can only find (in 24.x) that the data moved around, possibly making it sometimes unavailable, but no behaviour changes...

Reply:
More the latter, I think: https://github.com/bitcoin/bitcoin/pull/25923#discussion_r955740987, https://github.com/bitcoin/bitcoin/pull/25176#discussion_r882090128.